tests/volume.rc

```diff
@@ -427,6 +427,14 @@ function gf_check_file_opened_in_brick {
         fi
 }
 
+function gf_open_file_count_in_brick {
```

Comment:
This is not used since do-not-reopen-fd.t is not present but I suppose it is fine.